docs/design/uptime-reputation-calculation.md

```diff
@@ -0,0 +1,177 @@
+# Uptime reputation caculation
+
+## Abstract
+
+Having a fair, simple and understandable way to calculate the uptime of _storagenodes (henceforth SNs)_ is crucial for not causing unfair disqualifications, clarifying how _storagenode operators (henceforth SNOs)_ should administrate and maintain their _SN_ for being align with the network requirements and last but not least have a clear proof of disqualification for _SN_ getting disqualified due to too many uptime check failures.
+
+This design doc proposes a different way for calculating uptime check scores.
+
+## Background
+
+Currently our disqualification system is described in the [disqualification design doc](disqualification.md) which is based in the calculation of different _SN_ reputation scores as described in [node-selection design doc](node-selection.md).
+
+The problem with the current uptime _SN_ reputation score calculation is that it isn't based on the actual time that a node is being offline, by cause of to the nature of the chosen model, causing that an _SN_ will get disqualified due its number of uptime checks success and failure which implies that new nodes can get disqualified having less uptime check failures than veteran nodes.
```

Comment:
`vetted nodes`

Reply:
so it's based on percent failures/total right?

Reply:
No, I think this was my misunderstanding. The veteran here mean't nodes that have been in the network for a long time.

Reply:
But, yes, currently it's based roughly on failures/total, but with forgetting.